http/src/main/resources/reference.conf

```diff
@@ -442,14 +442,13 @@ gceClusterResources {
   jupyterDockerCompose = "jupyter-docker-compose-gce.yaml"
   gpuDockerCompose = "gpu-docker-compose.yaml"
   rstudioDockerCompose = "rstudio-docker-compose-gce.yaml"
-  proxyDockerCompose = "proxy-docker-compose.yaml"
-  welderDockerCompose = "welder-docker-compose.yaml"
+  proxyDockerCompose = "proxy-docker-compose-gce.yaml"
+  welderDockerCompose = "welder-docker-compose-gce.yaml"
   proxySiteConf = "cluster-site-gce.conf"
   jupyterNotebookConfigUri = "jupyter_notebook_config.py"
   jupyterNotebookFrontendConfigUri = "notebook.json"
   customEnvVarsConfigUri = "custom_env_vars.env"
-  cryptoDetectorDockerCompose = "crypto-detector-docker-compose.yaml"
-  networkDockerCompose = "network-gce.yaml"
```

Comment:
`network-gce.yaml` does not exist in Leo and is not referenced elsewhere, so removed it